### PR TITLE
DynamoDB persistence docs

### PR DESCRIPTION
**Pull request by timbussmann**

### CODE REVIEW

**Review by kentdr — APPROVED** [review]

File: samples/dynamodb/simple/DynamoDB_1/Client/Program.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.DynamoDB.Simple.Client";
+        var endpointConfiguration = new EndpointConfiguration("Samples.DynamoDB.Simple.Client");
+        endpointConfiguration.UsePersistence<LearningPersistence>();
```
Comment (kentdr) [inline samples/dynamodb/simple/DynamoDB_1/Client/Program.cs:11]: Should this be Dynamo?

**Review by kentdr — APPROVED** [review]

File: persistence/dynamodb/outbox.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+snippet: DynamoOutboxTableConfiguration
+
+NOTE: When using the same table for Saga and Outbox data, use the [shared table configuration API](/persistence/dynamodb/#usage-customizing-the-table-used) instead.
+
+The Outbox uses [time-to-live](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html) to ensure cleanup of successfully processed message after some time, therefore the Outbox table requires a `TableConfiguration.TimeToLiveAttributeName` value to be set. When installers are enabled, NServiceBus ensures the proper time-to-live configuration when the table is being created, otherwise users need to ensure that the table is properly configured.
+
+## Outbox cleanup
+
+When the Outbox processed a message, messages with the same message ID will be deduplicated for 7 days by default. To customize the deduplication timespan, use the following API:
```
Comment (timbussmann) [inline persistence/dynamodb/outbox.md:23]:
```suggestion
When the Outbox successfully processes a message, messages with the same message ID will be ignored (deduplicated) for 7 days by default. To customize the deduplication timespan, use the following API:
```

[the remark this answers is not in the document]
Comment (timbussmann) [reply persistence/dynamodb/outbox.md]: I slightly tweaked the suggestion, I hope you're good with that one too.

File: samples/dynamodb/simple/DynamoDB_1/Client/Program.cs (hunk 1 above)
Comment (timbussmann) [reply samples/dynamodb/simple/DynamoDB_1/Client/Program.cs:11]: the client doesn't need any persister, but there's not much value in configuring dynamo as the persister. however we should be able to remove the persistence config completely.

Comment (timbussmann) [reply samples/dynamodb/simple/DynamoDB_1/Client/Program.cs]: removed the persistence config from the client config completely

**Pull request merged**